setup.cfg

```diff
@@ -0,0 +1,25 @@
+[bumpversion]
+current_version = 3.4.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+search = version="{current_version}"
+replace = version="{new_version}"
+
+[bumpversion:file:bwapi/__init__.py]
+search = __version__ = "{current_version}"
+replace = __version__ = "{new_version}"
+
+[metadata]
+license_file = LICENSE.md
+
+[flake8]
+max-line-length = 88
```

Comment:
Why 88? I figure most people have high resolution screens that handle much longer lines these days, and usually go for 100 or 120 max length.

Reply:
That's just the default for the [black formatter](https://github.com/ambv/black#line-length). We don't have to use back and we can go back, or we can adjust it that parameter.  I find black has been helpful with readability and consistency.   Also some interesting stats on python line lengths [here](https://jakevdp.github.io/blog/2017/11/09/exploring-line-lengths-in-python-packages/)